packages/dart/lib/src/objects/parse_object.dart

```diff
@@ -162,7 +162,7 @@ class ParseObject extends ParseBase implements ParseCloneable {
       return response;
     } else {
       // if have network connection error
-      if ((response.error?.message ?? "") == "NetworkError") {
+      if ((response.error?.message ?? "").contains("NetworkError")) {
```

Comment:
Why is this parsing an error message, is there no error code to compare?

Reply:
I changed it, please check again